tests/php/sync/test_class.jetpack-sync-full.php

```diff
@@ -366,6 +366,154 @@ function test_full_sync_enqueue_term_relationships() {
 		$this->assertSame( $finished, true );
 	}
 
+	/**
+	 * This tests tries to test the accuracy of the total that gets stored in the enqueue status.
+	 * Since term relationships are not enqueued with an autoincromenting primary key.
+	 *
+	 */
+	function test_full_sync_enqueue_older_post_add_term_relationships() {
```

Comment:
Do we also need a test for when a term relationship has been updated? This is a rare case, but it can happen when a manual is involved intervention (migration, or a backup restore)

Reply:
You bring up a good point The `term_relationship` table is pretty simple. 
It is just 3 columns `object_id` and `taxonomy_term_id` and `term_order`.
as far as I know you can't change the `term_order` independently from `object_id` and `taxonomy_term_id` 

Since we are ordering things by `object_id` and then `taxonomy_term_id` even if the term_order changes we should be able to add that to the full sync before we send it. 

@lezama Do you know if we send the `term_order` during full sync. We seem to use in for diffing See https://github.com/Automattic/jetpack/blob/a4a4929c7dc3fa007a9e556deddeacfa1389bb5e/packages/sync/src/Defaults.php#L637